readme/Contrib.scalatex

```diff
@@ -0,0 +1,69 @@
+@import Main._
+@import scalaworld.Readme._
+
+@sect{Part 6 - Contrib}
+  Scala.meta contrib is a module that provides common utilities for handling
+  scala.meta data structures.
+
+  @p
+    To use contrib, @code{import scala.meta.contrib._}.
+
+  @p
+    Contrib exposes some collection-like methods on @code{Tree}.
+    @meta
+      source"""
+      class A
+      trait B
+      object C
+      object D
+      """.find(_.is[Defn.Object])
```

Comment:
It's an actual method, you can create custom "classifiers" like [here](https://github.com/scalameta/scalameta/blob/254ec8f81434214ed437f52213f852cd0a5069f1/scalameta/contrib/src/main/scala/scala/meta/contrib/TokenClasses.scala#L10-L16) and use the same syntax `.is[Trivia]`